CHANGELOG.md

```diff
@@ -51,6 +51,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 - **AWS SQS Scaler**: Support for scaling to include delayed messages. ([#4377](https://github.com/kedacore/keda/issues/4377))
 - **Governance**: KEDA transitioned to CNCF Graduated project ([#63](https://github.com/kedacore/governance/issues/63))
+- **Kafka Scaler**: Add UnsafeSSL flag for kafka scaler ([#4911](https://github.com/kedacore/keda/pull/4911))
```

Comment:
please move this to Improvements section and also link issue here, not PR please

Reply:
thanks @zroubalik this should be complete now;